package.json

```diff
@@ -1,15 +1,15 @@
 {
-  "name": "xircuits",
-  "version": "1.6.0",
+  "name": "tvb-ext-xircuits",
+  "version": "0.1a1",
   "description": "A JupyterLab extension for rendering and editing xircuit files.",
   "keywords": [
     "jupyter",
     "jupyterlab",
     "jupyterlab-extension"
   ],
-  "homepage": "https://github.com/XpressAI/xircuits",
+  "homepage": "https://github.com/the-virtual-brain/tvb-ext-xircuits",
   "bugs": {
-    "url": "https://github.com/XpressAI/xircuits/issues"
+    "url": "https://req.thevirtualbrain.org"
   },
   "license": "BSD-3-Clause",
```

Comment:
I see the license is Apache in LICENSE file, but here is BSD.
I wonder which one is the correct one ?